reana-ui/src/pages/workflowDetails/components/WorkflowFiles.js

```diff
@@ -91,8 +91,9 @@ export default function WorkflowFiles({ id }) {
       const fileExt = fileName.split(".").pop();
       content = `${fileExt} files cannot be previewed. Please use download.`;
     } else if (size > SIZE_LIMIT) {
-      content = `File size is too big to be previewed (limit ${SIZE_LIMIT /
-        1024 ** 2}MB). Please use download.`;
+      content = `File size is too big to be previewed (limit ${
+        SIZE_LIMIT / 1024 ** 2
```

Comment:
`SIZE_LIMIT` seems like a good candidate to be moved to the new configuration, doesn't it?

Reply:
Yes, definitely.